source/slang-record-replay/record/slang-module.cpp

```diff
@@ -213,6 +213,16 @@ namespace SlangRecord
         return res;
     }
 
+    SLANG_NO_THROW SlangResult ModuleRecorder::precompileForTarget(
+        SlangCompileTarget target,
+        ISlangBlob** outDiagnostics)
+    {
+        // TODO: We should record this call
```

Comment:
There needs to be an issue associated with the TODO.

Reply:
Added